packages/ui/src/containers/AuthContainer.ts

```diff
@@ -159,8 +159,22 @@ export class AuthContainer {
     let form = this.accountForm!;
     if (form instanceof NewAccountFormData && form.clean()) {
       // Save the private and public keys to disk.
-      saveToFile(form.privateKeyBase64, `${form.name.$}.private.key`);
-      saveToFile(form.publicKeyBase64.$!, `${form.name.$}.public.key`);
+      saveToFile(
+        Keys.Ed25519.privateKeyEncodeInPem(decodeBase64(form.privateKeyBase64)),
+        `${form.name.$}-account-private.pem`
```

Comment:
The Rust client is naming this file `secret_key.pem`.  The main thing I'd like to see is consistent naming of secret keys - both in the codebase and in all user-facing situations (docs/files/GUIs), but for now using "secret" instead of "private" in the generated filename would be a start.

Maybe that's work for a separate ticket though?